lib/js/components/SelectionTile/SelectionTile.vue

```diff
@@ -0,0 +1,258 @@
+<template>
+	<div
+		class="selectionTile"
+		:class="[
+			{
+				'-selected': isSelected,
+				'-default': state === SELECTION_TILE_STATE.DEFAULT,
+				'-disabled': state === SELECTION_TILE_STATE.DISABLED,
+				'-loading': state === SELECTION_TILE_STATE.LOADING,
+			},
+		]"
+		@click="updateIsSelected(!isSelected)"
+	>
+		<div class="selectionTile__wrapper">
+			<div class="selectionTile__selectionControl">
+				<radio-button
+					v-if="type === SELECTION_TILE_TYPE.RADIO_BUTTON"
+					:size="RADIO_BUTTON_SIZE.X_SMALL"
+					:is-selected="isSelected"
+					:state="selectionControlState"
+					@update:isSelected="updateIsSelected($event)"
+				/>
+				<checkbox
+					v-else
+					:size="CHECKBOX_SIZE.X_SMALL"
+					:is-selected="isSelected"
+					:state="selectionControlState"
+					@update:isSelected="updateIsSelected($event)"
+				/>
+			</div>
+			<div class="selectionTile__textWrapper">
+				<div class="selectionTile__title">{{ title }}</div>
+
+				<div v-if="supportingText" class="selectionTile__supportingText"
+					>{{ supportingText }}
+				</div>
+			</div>
+		</div>
+		<div
+			v-if="icon || isLoading"
+			class="selectionTile__icon"
+			@click.stop="isLoading ? null : $emit('icon-click')"
+		>
+			<icon
+				:icon="isLoading ? ICONS.FAD_SPINNER_THIRD : icon"
+				:size="ICON_SIZES.X_SMALL"
+				:spinning="state === SELECTION_TILE_STATE.LOADING"
```

Comment:
```suggestion
				:spinning="isLoading"
```
to make it consisten with consition in `:icon`